lang/wasm.cc

```diff
@@ -290,51 +195,60 @@ struct from_val_visitor {
             "f32",
             "f64",
             "v128",
-            "externref",
             "funcref",
+            "externref",
```

Comment:
Why was the order changed here?

Reply:
It matched the order of enum elements in `lang/wasmtime.hh` https://github.com/scylladb/scylladb/blob/41b91e36324fbd373d3fe6ca1006342f663e2568/lang/wasmtime.hh#L492-L497 and now it matches the order in the wasmtime crate https://docs.rs/wasmtime/latest/wasmtime/enum.Val.html